Gemfile

```
@@ -38,6 +38,7 @@ group :development, :test do
   gem 'annotate', '~> 3.2', '>= 3.0.3'
   gem 'dotenv-rails', '~> 2.7.6'
   gem 'factory_bot_rails', '~> 5.1', '>= 5.1.1'
+  gem 'knapsack', '~> 4.0.0', '>= 4.0.0'
```

```suggestion
  gem 'knapsack', '~> 4.0'
```

maybe that's better so we allow any 4.x version?